Config.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
```

Comment:
This is unnecessary, and should be removed. The name should ideally be changed to remove the `.in`, but I believe we are limited to other CARMA CMake modules.

Reply:
I think this is all kind of based on this [tutorial](https://cmake.org/cmake/help/latest/guide/tutorial/Adding%20Export%20Configuration.html). It seems to match pretty closely with what is implemented in the carma-builds InstallConfigs.cmake . This tutorial seems to be up to date with the current CMake verison. Are we doing something else that makes this unnecessary?

Reply:
Using the configuration we have now is a bit over-complicating things. `*Config.cmake.in` is when you have more elaborate dependency resolution. I have needed to use `*Config.cmake.in` only once or twice. Refactoring this makes the `*Config.cmake` install a simple `install(FILES)` call.

Reply:
It sounds like this is more of a preference on how this process should be setup rather than a case in which we are doing something wrong or incorrect. If this is the case I think having it consistent with our other CMake installation setups is better for right now. We can put an issue up for changing this process if there is some substantial benefit and pick it up if necessary.

Reply:
That makes sense. This is indeed a case where it's not wrong but could be improved. We'll punt this to separate task/issue.